platform/jewel/ui/src/main/kotlin/org/jetbrains/jewel/ui/component/IconButton.kt

```diff
@@ -96,6 +97,27 @@ public fun IconButton(
     )
 }
 
+@Composable
+public fun BackgroundlessIconButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    focusable: Boolean = true,
+    style: IconButtonStyle = JewelTheme.iconButtonStyleWithoutBackground,
+    interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
+    content: @Composable (BoxScope.(IconButtonState) -> Unit),
+) {
+    IconButton(
+        onClick = onClick,
+        modifier = modifier,
+        enabled = enabled,
+        focusable = focusable,
+        style = style,
+        interactionSource = interactionSource,
+        content = content,
+    )
+}
+
 @Composable
 public fun SelectableIconButton(
```

Comment:
I'm wondering whether we should have selectable and toggleable background-less variants, but most likely, no. You still need a bg to show when they're selected/checked, and they do not have a bg otherwise...